### ISSUE TITLE
generate:controller --actions not parsed.

### ISSUE THREAD

**Issue by leevigraham** [sensiolabs/SensioGeneratorBundle#350]
Using either examples in the docs:

```` bash
$ php app/console generate:controller --actions="showPostAction:/article/{id} getListAction:/_list-posts/{max}:AcmeBlogBundle:Post:list_posts.html.twig"
$ php app/console generate:controller --actions=showPostAction:/article/{id} --actions=getListAction:/_list-posts/{max}:AcmeBlogBundle:Post:list_posts.html.twig
````

throws an exception:

```` bash
[Symfony\Component\Debug\Exception\ContextErrorException]  
Warning: Illegal string offset 'name'
````

Exception Trace:

````
Exception trace:
 () at /Volumes/Sites/Work/Internal/verismo2/vendor/sensio/generator-bundle/Sensio/Bundle/GeneratorBundle/Generator/ControllerGenerator.php:56
 Symfony\Component\Debug\ErrorHandler->handleError() at /Volumes/Sites/Work/Internal/verismo2/vendor/sensio/generator-bundle/Sensio/Bundle/GeneratorBundle/Generator/ControllerGenerator.php:56
 Sensio\Bundle\GeneratorBundle\Generator\ControllerGenerator->generate() at /Volumes/Sites/Work/Internal/verismo2/vendor/sensio/generator-bundle/Sensio/Bundle/GeneratorBundle/Command/GenerateControllerCommand.php:102
 Sensio\Bundle\GeneratorBundle\Command\GenerateControllerCommand->execute() at /Volumes/Sites/Work/Internal/verismo2/vendor/symfony/symfony/src/Symfony/Component/Console/Command/Command.php:259
 Symfony\Component\Console\Command\Command->run() at /Volumes/Sites/Work/Internal/verismo2/vendor/symfony/symfony/src/Symfony/Component/Console/Application.php:886
 Symfony\Component\Console\Application->doRunCommand() at /Volumes/Sites/Work/Internal/verismo2/vendor/symfony/symfony/src/Symfony/Component/Console/Application.php:195
 Symfony\Component\Console\Application->doRun() at /Volumes/Sites/Work/Internal/verismo2/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Console/Application.php:96
 Symfony\Bundle\FrameworkBundle\Console\Application->doRun() at /Volumes/Sites/Work/Internal/verismo2/vendor/symfony/symfony/src/Symfony/Component/Console/Application.php:126
 Symfony\Component\Console\Application->run() at /Volumes/Sites/Work/Internal/verismo2/app/console:27
````

It looks like `GenerateControllerCommand::parseActions()` is always being passed an array when triggers an early return.

Removing `InputOption::VALUE_IS_ARRAY` from the GenerateControllerCommand actions option and passing a string to `--actions` generates the controller as expected.

**Comment by javiereguiluz** [comment]
I'm closing it because this bug was fixed by #385, which was merged today.

**Comment by leevigraham** [comment]
:+1:
